src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -69,13 +70,20 @@
 	private static final String[] OPERATIONS_ROLES = {"operations-user"};
 	
 	public static final String DEFAULT_INDEX_FIELD = "defaultIndex";
+
+    private final IndexMappingLoader mappingLoader;
+	
+	@Inject
+	public KibanaSeed(IndexMappingLoader loader) {
```

Comment:
`@Inject` is magic; this constructor will be called when we are expecting it in the DynamicACL constructor